auto-value-parcel/src/test/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtensionTest.java

```diff
@@ -467,6 +469,224 @@
         .generatesSources(expectedNotMatching, expectedMatching);
   }
 
+  @Test public void writeToParcelOmittedWhenAlreadyDefined() {
+    JavaFileObject notMatching = JavaFileObjects.forSourceString("test.No", ""
+            + "package test;\n"
+            + "import android.os.Parcel;\n"
+            + "import android.os.Parcelable;\n"
+            + "import com.google.auto.value.AutoValue;\n"
+            + "@AutoValue public abstract class No implements Parcelable {\n"
+            + "  public abstract String name();\n"
+            + "  @Override public abstract void writeToParcel(Parcel parcel, int flags);\n"
```

Comment:
Looks like this has both an abstract version and an implemented version.  And the implemented version doesn't match the interface.